pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -756,7 +756,7 @@ public void testDeleteNamespaces() throws Exception {
 
         mockZooKeeperGlobal.delete("/admin/partitioned-topics/" + topicName.getPersistenceNamingEncoding(), -1);
 
-        testNs = this.testGlobalNamespaces.get(0);
+        testNs = this.testLocalNamespaces.get(2);
```

Comment:
I change here, because we tried to delete non-persistent topics which previous don't.
When we get full of non-persistent, we should try to get from non-global namespace.